tests/integration/test_case.go

```diff
@@ -593,3 +593,35 @@ type BackupImport struct {
 	// contains this string.
 	ExpectedError string
 }
+
+// IsNetworkAction returns true if the given action involves the network subsystem.
+func IsNetworkAction(act any) bool {
+	switch act.(type) {
+	case ConfigureNode:
```

Comment:
suggestion: I don't think you need to check all of these?  `ConfigureNode` is required for all P2P tests IIRC.  Expecting all future network test actions to consistently be added to this switch seems a little optimistic to me and I would rather avoid the need if we can. 

suggestion: If keeping all the actions in the switch, I think collapsing them into a single case might be easier to read:
```
switch act.(type) {
	case ConfigureNode,
	        ConnectPeers,
	        ConfigureReplicator,
	        etc...:
	      return true
}
```

Reply:
I simplified it to just check for `ConfigureNode`.